src/pymor/algorithms/timestepping.py

```diff
@@ -69,8 +70,9 @@ def solve(self, initial_time, end_time, initial_data, operator, rhs=None, mass=N
         operator
             The |Operator| A.
         rhs
-            The right-hand side F (either |VectorArray| of length 1 or |Operator| with
-            `source.dim == 1`). If `None`, zero right-hand side is assumed.
+            The right-hand side F (either |VectorArray| that might be converted into a
+            suitable |SelectionOperat| or |Operator| with `source.dim == 1`).
+            If `None`, zero right-hand side is assumed.
```

Comment:
I think a bit more explanation is required. Maybe:

```suggestion
            The right-hand side. Either a vector-like |Operator| with `source.dim == 1` or a |VectorArray|.
            If the |VectorArray| contains more than one vector,  
            :func:`~pymor.operators.constructions. vector_array_to_selection_operator` is used to convert
            the array to an |Operator| that is piecewise constant in time. If `None`, zero right-hand side is 
            assumed.
```

?